### ISSUE TITLE
option "--monitor=right" ignored after update from 1.6.0 to 1.8.0

### ISSUE THREAD

**Issue by cobour** [prb28/vscode-amiga-assembly#258]
I used the extension v1.6.0 with these options in launch.json:

                "--fullscreen=1",
                "--monitor=right"

The fullscreen-option is still working, but the monitor-option is ignored (due to new fork of fs-uae?).

Issue found on macOS 13.2.1 using VSC 1.76.1 (and extension 1.8.0).

**Comment by prb28** [comment]
should work according to : https://github.com/FrodeSolheim/fs-uae/blob/15a51e33916f87b4d74510ad99ed7297f5f3f18a/docs/options/monitor
@grahambates what do you think about this issue?

**Comment by grahambates** [comment]
My guess is that it's not yet implemented in FS-UAE 4.x, which the new fork is based on. This is a pre-release version and may still be missing some features. I'll try it out on the official build to confirm.

If this (or any other missing feature) is a show-stopper, we could look at porting the Barto debugger implementation to 3.X but I'm guessing this will be a lot more effort. I was hoping to avoid this in order to keep the patches for FS-UAE and WinUAE consistent. It also would prevent us taking advantage of the profiler tools, as this relies on internal data structures only present in 4.2+.

**Comment by cobour** [comment]
In my opinion this is NOT a show-stopper, it is just a bit annoying. I now use the window mode again for the launchers and wait for a newer version that supports the monitor option again correctly.
Thanks for your amazing work on this extension! I love to use it...

**Comment by prb28** [comment]
I agree, it's better to wait a stable 4 release and do not create a 3 back port. The old versions of the extension are still accessible.

**Comment by grahambates** [comment]
I can confirm that the issue is present in the non-patched FS-UAE 4.x.
